<h1 id="publications">Publications</h1>
<p>van Dijk D*, <strong>Gigante S</strong>*, Strzalkowski A, Wolf G<span class="math inline">.<sup>#</sup></span>, Krishnaswamy S<span class="math inline">.<sup>#</sup></span>. Modeling Dynamics with Deep Transition-Learning Networks. arXiv preprint <a href="https://arxiv.org/abs/1802.03497">arXiv:1802.03497</a>. 2018 Feb 10.</p>
<p>Moon KR*, van Dijk D*, Wang Z*, <strong>Gigante S</strong>, Burkhardt D, Chen W, van den Elzen A, Hirn MJ, Coifman RR, Ivanova NB<span class="math inline">.<sup>#</sup></span>, Wolf G<span class="math inline">.<sup>#</sup></span>, Krishnaswamy S<span class="math inline">.<sup>#</sup></span>. Visualizing transitions and structure for high dimensional data exploration. bioRxiv. 2017 Jan 1:120378. DOI: <a href="https://doi.org/10.1101/120378">10.1101/120378</a>.</p>
<p><strong>Gigante S</strong>. Picopore: a tool for reducing the storage size of Oxford Nanopore Technologies datasets without loss of functionality. F1000Research. 2017;6. DOI: <a href="https://doi.org/10.12688/f1000research.11022.3">10.12688/f1000research.11022.3</a></p>
<h1 id="invited-talks">Invited Talks</h1>
<p>Nanopore Community Meeting, Oxford Nanopore Technologies. Direct Determination of Genome-Wide, Allele-specific Mouse DNA Methylation from Nanopore Long-Read Sequencing, New York, 1 December 2017. <a href="https://vimeo.com/250463649">Video</a>, <a href="https://nanoporetech.com/about-us/news/nanopore-community-meeting-day-2-writeup-0">Report</a></p>
<p>Monash Bioinformatics Platform Seminar, Monash University. Simultaneous Haplotyping and DNA Base Modification Detection with Nanopore Sequencing, Melbourne, 16 August 2017.</p>
<p>Bioinformatics Division Seminar, Walter &amp; Eliza Hall Institute of Medical Research. Simultaneous Whole Genome Sequencing and DNA Base Modification Detection with Nanopore Sequencing, Melbourne, 11 July 2017.</p>
<p>London Calling Conference Software Demonstration, Oxford Nanopore Technologies. Picopore: A tool for reducing the storage size of Oxford Nanopore Technologies datasets without loss of functionality, London, 4 May 2017.</p>
<p>London Calling Conference Lightning Talk, Oxford Nanopore Technologies. In-house training of the nanonet local basecaller: opportunities and challenges, London, 5 May 2017. <a href="https://vimeo.com/218020005">Video</a></p>
<h1 id="posters">Posters</h1>
<p><strong>Gigante S</strong>*, van Dijk D*, Strzalkowski A, Wolf G<span class="math inline">.<sup>#</sup></span>, Krishnaswamy S<span class="math inline">.<sup>#</sup></span>. Modeling Dynamics with Deep Transition-Learning Networks. 12th Annual Machine Learning Symposium, The New York Academy of Sciences. New York, 9 March 2018.</p>
<p><strong>Gigante S</strong>, Lucattini A, Keniry A, Gouil Q, Tinning M, Gordon L, Woodruff C, Ritchie M, Speed T, Blewitt M. Direct Determination of Genome-Wide, Allele-specific Mouse DNA Methylation from Nanopore Long-Read Sequencing. Nanopore Community Meeting, Oxford Nanopore Technologies. New York, 1 December 2017.</p>
<p>Woodruff C, <strong>Gigante S</strong>, Lucattini A, Keniry A, Ritchie M, Speed T. Can Nanopore Sequencing Change Cancer Diagnostics?. VCCC Postdoctoral Symposium: New Frontiers in Cancer Research, Victorian Comprehensive Cancer Centre. Melbourne, 26 May 2017. DOI: <a href="https://doi.org/10.13140/RG.2.2.15549.05601">10.13140/RG.2.2.15549.05601</a></p>
<p><strong>Gigante S</strong>, Woodruff C, Lucattini A, Ritchie M, Speed T. In-house training of the nanonet local basecaller: opportunities and challenges. London Calling Conference, Oxford Nanopore Technologies. London, 5 May 2017. DOI: <a href="http://dx.doi.org/10.7490/f1000research.1114122.1">10.7490/f1000research.1114122.1</a></p>
<h1 id="press">Press</h1>
<p>Murray, H. Nanopore's yottasteps. <a href="https://blog.f1000.com/2017/03/20/nanopores-yottasteps/">F1000Research blog</a>, 20 March 2017.</p>
